regress/pesterTests/Authorized_keys_fileperm.Tests.ps1

```diff
@@ -27,8 +27,9 @@ Describe "Tests for authorized_keys file permission" -Tags "CI" {
         $opensshbinpath = $OpenSSHTestInfo['OpenSSHBinPath']
         Remove-Item -Path (Join-Path $testDir "*$sshLogName") -Force -ErrorAction SilentlyContinue        
         $platform = Get-Platform
-        $skip = ($platform -eq [PlatformType]::Windows) -and ($PSVersionTable.PSVersion.Major -le 2)
-        if(($platform -eq [PlatformType]::Windows) -and ($psversiontable.BuildVersion.Major -le 6))
+        #skip on win7 because Set-Acl failed due to the task schedular (*-ScheduledTask) cmdlets does not exist on win7 and win8        
+        $skip = ($platform -eq [PlatformType]::Windows) -and ([Environment]::OSVersion.Version.Major -le 6) -and ([Environment]::OSVersion.Version.Minor -le 2)
```

Comment:
Is it And / OR for the last two conditions
([Environment]::OSVersion.Version.Major -le 6) -or ([Environment]::OSVersion.Version.Minor -le 2)

Reply:
and. 6.2 is win8. 6.1 is win7